packages/context-selector/src/index.tsx

```diff
@@ -0,0 +1,11 @@
+export * from './Dropdown';
+export * from './DropdownProvider';
+export * from './ContextSelector';
+export * from './useDropdownProviderRef';
+
+export {
+  SearchableDropdownElement as HTMLSearchableDropdownCustomElement,
+  SearchableDropdownProviderElement as HTMLSearchableDropdownProviderCustomElement,
+} from '@equinor/fusion-wc-searchable-dropdown';
+
```

Comment:
Please expose all types needed
 SearchableDropdownResultItem, SearchableDropdownResult 

same shoud be done in @equinor/fusion-wc-searchable-dropdown

Reply:
sorry seems to be done in @equinor/fusion-wc-searchable-dropdown <3